tests/testapp/views.py

```diff
@@ -55,21 +54,46 @@ class StudentViewSet(viewsets.ModelViewSet):
 	queryset = Student.objects.all()
 
 
+class StudentRestQLViewSet(RestQLViewMixin, viewsets.ModelViewSet):
+	serializer_class = StudentSerializer
+	queryset = Student.objects.all()
+	restql_orm_mapping = {
+		"select": {
+			"course": "course"
+		},
+		"prefetch": {
+			"course": {
+				"nested": {
+					"books": "course__books"
+				}
+			}
+		}
+	}
```

Comment:
I think that's a lot cleaner, but these are dictionaries so that they match up with what fields are queried for. For example, if we names the serializer field `course` to instead be `class`:

```python
class = CourseSerializer(source="course", many=False, read_only=True)
```

We won't know what to prefetch/select for it without specifying. This also goes for method fields, where we might use data we want to prefetch or select.

Reply:
Admittedly.. `class` is a poor name choice since it's a reserved word, lol. I'll add an aliased field on student to illustrate this in the test.